engine/table/src/main/java/io/deephaven/stream/StreamToBlinkTableAdapter.java

```diff
@@ -277,50 +212,49 @@ public Table table() {
 
     @Override
     public void close() {
-        if (!alive) {
-            return;
-        }
-        synchronized (this) {
-            if (!alive) {
-                return;
-            }
-            alive = false;
```

Comment:
I'm assuming this old pattern works, you just wanted something a bit prettier?

Reply:
I thought it would be better to eliminate the lock call.